nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -294,6 +294,8 @@
 
 - DocBook option documentation is no longer supported, all module documentation now uses markdown.
 
+- `outline` can now be configured to use local filesystem storage instead of S3 storage.
```

Comment:
```suggestion
- `services.outline` can now be configured to use local filesystem storage instead of S3 storage using [services.outline.storage.storageType](#opt-services.outline.storage.storageType).
```